data/variabledefs-rescue.csv

```diff
@@ -0,0 +1,136 @@
+gas,sector,unit,include_in_total,griddinglevel,proxy_path,proxy_name,proxy_sector

```

Comment:
I know this is from early days, but should we call this `proxy_name` or rather `output_file`? I've lost a bit the thread which of the columns are related to reading in the data versus writing it out.

Reply:
Ok, rewrote to `output_variable`